src/cairo_threads.rs

```diff
@@ -163,8 +164,8 @@ fn draw_initial(format: Format, width: i32, height: i32) -> (Box<[u8]>, i32) {
     (buf, image.get_stride())
 }
 
-fn draw_slow(cr: &Context, delay: u32, x: f64, y: f64, radius: f64) {
-    thread::sleep_ms(delay);
+fn draw_slow(cr: &Context, delay: u64, x: f64, y: f64, radius: f64) {
```

Comment:
If the standard library "suggests" we take advantage of richer types i.e. adopt the more specific `Duration` type it seems logical to do this fully.